examples/weather-alert/main.bal

```diff
@@ -0,0 +1,81 @@
+};
+
+sns:Client amazonSNSClient = check new(config);
+
+public function main() returns error? {
+    // First we would need to create separate topics for each city. If a topic with the same name already exists,
+    // the existing topic will be returned
+    string colombo = check amazonSNSClient->createTopic("Colombo");
+    string london = check amazonSNSClient->createTopic("London");
+    string washington = check amazonSNSClient->createTopic("Washington");
+
+    // If you create a subscription for an email address, you need to confirm the subscription by clicking on the
+    // link sent to your email
+    _ = check amazonSNSClient->subscribe(colombo, yourEmail, sns:EMAIL);
+    _ = io:readln("Please confirm the subscription to the Colombo topic by clicking on the link sent to your email and press any key to continue.");
+
+    _ = check amazonSNSClient->subscribe(washington, yourEmail, sns:EMAIL);
+    _ = io:readln("Please confirm the subscription to the Washington topic by clicking on the link sent to your email and press any key to continue.");
+
+    // If your SNS account is in the sandbox mode, you need to verify your phone number before subscribing to a topic
+    _ = check amazonSNSClient->createSMSSandboxPhoneNumber(yourPhone);
+    string otp = io:readln("Please verify the phone number by entering the OTP sent to your mobile: ");
+    _ = check amazonSNSClient->verifySMSSandboxPhoneNumber(yourPhone, otp);
+
+    // You can create a subscription for a phone number without confirming it
+    _ = check amazonSNSClient->subscribe(london, yourPhone, sns:SMS);
+    _ = check amazonSNSClient->subscribe(washington, yourPhone, sns:SMS);
+
+    // You can publish the a message to a topic. The same message will be sent to all the subscribers of the topic.
+    // You should see the message in your email and SMS inbox
+    _ = check amazonSNSClient->publish(colombo, "The temperature in Colombo is 24 degrees Celsius");
+    _ = check amazonSNSClient->publish(london, "The temperature in London is 18 degrees Celsius");
+
+    // You can also publish different messages to different subscribers of the same topic. The message delivered 
+    // to a subscriber depends on the protocol of the subscription
+    _ = check amazonSNSClient->publish(washington, {
+        "default": "The temperature in Washington is 30 degrees Celsius",
+        "sms": "Hello! The temperature in Washington is 30 degrees Celsius",
+        "email": "Good morning! The temperature in Washington is 30 degrees Celsius"
+    });
+
+    // Finally you can list all subscriptions for the 3 topics and unsubscribe from them
+    _ = check from sns:Subscription subscription in amazonSNSClient->listSubscriptions()
+        where subscription.topicArn == colombo || 
+                subscription.topicArn == london || 
+                subscription.topicArn == washington
+        do {
+            check amazonSNSClient->unsubscribe(subscription.subscriptionArn);
+        };
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in [f10a29e](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/51/commits/f10a29e567ead1ff0951b586cee691aafcc0e34f)